src/main/java/org/example/Methods.java

```diff
@@ -0,0 +1,179 @@
+package org.example;
+
+
+import java.util.*;
+import java.util.function.Predicate;
+
+public class Methods {
+
+    public static void State(int number, String player, String[][] board) {
+        switch (number) {
+            case 1 : {
+                board[0][0] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+
+            case 2 :{
+                board[0][1] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 3 : {
+                board[0][2] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 4 : {
+                board[1][0] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 5 : {
+                board[1][1] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 6 : {
+                board[1][2] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 7 : {
+                board[2][0] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 8 : {
+                board[2][1] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            case 9 : {
+                board[2][2] = player;
+                System.out.println(Arrays.deepToString(board));
+                break;
+            }
+            default:
+                System.out.println("koniec gry");
+            break;
+        }
+    }
+    public static boolean CheckedLists(int fieldNumber, List<Integer> listNumbers){
+        Integer IntegerNumber = fieldNumber;
+        boolean b = false;
```

Comment:
to mi sie nie podoba :D co to jest b i czemu trzeba uzywac break :D to robi zwykly equals z listy nie ? to moze jakis stream.findFirst czy cos :D?